pkg/migration/generate.go

```diff
@@ -35,6 +35,34 @@ const (
 	csiHealthMonitorControllerContainerName = "csi-health-monitor-controller"
 )
 
+var (
+	builtinPxVolumeNames = map[string]bool{
+		// names from DaemonSet portworx container
+		"containerddir": true,
+		"criosock":      true,
+		"dbusmount":     true,
+		"dev":           true,
+		"diagsdump":     true,
+		"dockersock":    true,
+		"etcpwx":        true,
+		"journalmount1": true,
+		"journalmount2": true,
+		"optpwx":        true,
+		"procmount":     true,
+		"sysdmount":     true,
+		// extra names from Operator portworx container to avoid conflicts
```

Comment:
it's unlikely the daemonset custom volume will use the same name, as the name is given by user. can we compare by the volume content?